common/djangoapps/student/management/tests/test_bulk_change_enrollment.py

```diff
@@ -90,21 +90,6 @@ def test_bulk_convert_with_org(self, from_mode, to_mode, mock_tracker):
                 CourseEnrollment.objects.get(mode=to_mode, course_id=course.id, user=user)
                 self._assert_mode_changed(mock_tracker, course, user, to_mode)
 
-    def test_with_org_and_course_key(self):
```

Comment:
This check removed in this [PR](https://github.com/edx/edx-platform/commit/ce50c9e620405b5452220fa0965200ecab67a491#diff-e10a709d51e1cc141a6ee6e31bdbbee2L75)

Reply:
This test is still necessary. They just took advantage of argparses mutually_exclusive_group feature to do the check for them, rather than do it in code. [link to argparse method](https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/management/commands/bulk_change_enrollment.py#L39-L46)